example/tests/test_blog.py

```diff
@@ -4,19 +4,19 @@
 
 
 def test_index(client, auth):
-    response = client.get("/")
+    response = client.get("blog/index")
     assert b"Log In" in response.data
     assert b"Register" in response.data
 
     auth.login()
-    response = client.get("/")
+    response = client.get("blog/index")
     assert b"test title" in response.data
     assert b"by test on 2018-01-01" in response.data
     assert b"test\nbody" in response.data
-    assert b'href="/1/update"' in response.data
+    assert b'href="blog/1/update"' in response.data
```

Comment:
I updated the href assert here, but it didn't work.  Looking closer at this test, I wasn't sure if there was a post "edit/update" done here.  It looks like just a auth.login() login before going to "blog/index".  I wanted to ask, in case I forgot a path change somewhere...

Reply:
I'm messing around with the tests myself and it does seem weird it's not asserting correctly? Running your branch locally I don't see nay problems so I am sure it's in the test configuration. I wouldn't worry about it too much honestly.